herddb-core/src/main/java/herddb/sql/DDLSQLPlanner.java

```diff
@@ -653,7 +654,9 @@ private Statement buildExecuteStatement(String defaultTableSpace, Execute execut
                 Object tableSpaceName = resolveValue(execute.getExprList().getExpressions().get(0), true);
                 String leader = null;
                 Set<String> replica = new HashSet<>();
-                int expectedreplicacount = 1;
+                int expectedreplicacount = this.manager.getServerConfiguration() != null
```

Comment:
Should we abstract this logic this.manager.getServerConfiguration() ! = null into some class or method and have a utility method to which we just pass property key 'PROPERTY_DEFAULT_REPLICA_COUNT' (in this case). We can reuse this functionality someplace else so we don't duplicate the code.

Reply:
It is a good suggestion.
I will change this line.
But I prefer not to change other lines in the scope of this patch.

I will double check, maybe ServerConfiguration is not expected to be null

Reply:
I have double checked and ServerConfiguration is "final" in DBManager so it is safe to consider it not null.
I have simplified this line.

Good catch @amitvc !